package.json

```diff
@@ -29,7 +31,8 @@
         "rimraf": "^2.6.2",
         "shortid": "^2.2.14",
         "video-react": "^0.13.2",
-        "vott-ct": "^2.1.17"
+        "vott-ct": "^2.1.11",
```

Comment:
This shouldn't downgrade the canvas tools library. This could be the cause of the compilation errors. Npm install should fix this

Reply:
Thank you!